src/Tag/driver.js

```diff
@@ -0,0 +1,13 @@
+export default class TagDriver
+{
+    constructor( wrapper )
+    {
+        this.wrapper = wrapper;
+    }
+
+    clickClose()
+    {
+        this.wrapper.find( '.tag__delete' ).first().simulate( 'click' );
```

Comment:
We shouldn’t use hard-coded classnames because our classnames are generated by webpack/css-loader and subject to change. Here we should use `cssMap.delete` instead